llvm/test/CodeGen/AMDGPU/memcpy-libcall.ll

```diff
@@ -0,0 +1,973 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx908 %s -o - | FileCheck %s
+
+%struct.S = type { [32 x i32] }
+
+@shared = addrspace(3) global %struct.S undef, align 4
+
+; Function Attrs: minsize
+define void @_Z12copy_genericPvPKv__minsize(ptr nocapture noundef writeonly %dest, ptr nocapture noundef readonly %src) #0 {
+; CHECK-LABEL: _Z12copy_genericPvPKv__minsize:
+; CHECK:       ; %bb.0: ; %entry
```

Comment:
Can you precommit this test so we can see before/after?

Reply:
before: crash Lol

Reply:
Oops, I should have read the commit message first, my bad
![](https://cdn3.emoji.gg/emojis/8637-sadcat-ok.png)